### PR TITLE
removing typos

### PR DESCRIPTION
**Pull request by ghost**
updated the docs little bit, added punctuation removed grammatical errors.

### CODE REVIEW

File: docs/src/pages/docs/api.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: API
 
 ## Memoize your Props!
 
-The React Charts `<Chart>` component has a few options that need to be **stable** or **memoized using either `React.useMemo` or `React.useCallback`**. Using an unstable option incorrectly shouldn't severly break any basic functionality, but could results in infinite change-detection loops in your app or at the very least, your charts will be severly non-performant. If an option says it needs to be stable, it's not kidding around!
+The React Charts `<Chart>` component has a few options that needs to be **stable** or **memoized using either `React.useMemo` or `React.useCallback`**. Using an unstable option incorrectly shouldn't severly break any basic functionality, but could results in infinite change-detection loops in your app or at the very least, your charts will be severly non-performant. If an option says it needs to be stable, it's not kidding around!
```
Comment (tannerlinsley) [inline docs/src/pages/docs/api.md:10]: This is incorrect. Go ahead and leave this one as is

File: docs/src/pages/docs/api.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ const data = [
 ]
 ```
 
-Visualization data can come in practically any shape and size, so **memoization of data into this shape is almost always necessary**.
+Visualization of data can come in practically any shape and size, so **memoization of data into this shape is almost always necessary**.
```
Comment (tannerlinsley) [inline docs/src/pages/docs/api.md:31]: Also not correct

File: docs/src/pages/docs/api.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ The individual datums in a series' `data` array can be anything you want! Just r
 The Chart component props can be passed in its `options` property object:
 
 ```javascript
-<Chart
+<
```
Comment (tannerlinsley) [inline docs/src/pages/docs/api.md:66]: Incorrect

File: docs/src/pages/docs/api.md [hunk 4]
```diff
@@ -126,12 +126,11 @@ const secondaryAxes = React.useMemo(
...
 ```
 
-The `initialHeight` and `initialWidth` expect a number, a default value is applied for each of those, 300 and 200 respectively. It's important to mention that these options are available SSR only.
-If you'd like to have a custom height and width in the client side you may have a wrapper div that sets the width and height CSS attributes
+The `initialHeight` and `initialWidth` expect a number, a default value is applied for each of those, 300 and 200 respectively. It's important to mention that these options are available for SSR only.
+If you'd like to have a custom height and width in the client side you may have a wrapper div that sets the width and height CSS attributes.
 
 `interactionMode` expect an string wich can be "primary" or "closest". It's been using for the tooltip position. By default, primary is being set.
 
-`showVoronoi` expect a boolean, it's a debug option to visualize the interaction click-map that sits on top of the chart.
```
Comment (tannerlinsley) [inline docs/src/pages/docs/api.md:134]: Leave this please

**Comment by tannerlinsley** [comment]
Too many errors and false positives. Sorry!

**Comment by ghost** [comment]
Hi @tannerlinsley,
in my next pull can you approve this ? 
image

**Pull request closed without merging**